commands/ls-id.go

```diff
@@ -10,9 +10,16 @@ import (
 
 func runLsID(cmd *cobra.Command, args []string) error {
 
-	var backend *cache.RepoCache
+	backend, err := cache.NewRepoCache(repo)
+	if err != nil {
+		return err
+	}
+	defer backend.Close()
```

Comment:
You should register a cleaner on interruption as well.

`interrupt.RegisterCleaner(backend.Close)`